spring-cloud-netflix-core/src/test/java/feign/httpclient/ProtobufSpringEncoderTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2012-2013 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feign.httpclient;
```

Comment:
These packages need to change to similar spring cloud packages

Reply:
It's so hard to change this because 'ApacheHttpClient` is a final class and `toHttpUriRequest' is not a public method, unless using reflect to invoke this method.

What’s your point of view on this?